File: javascript/EliminacionPersonalizada.js

let numeros = [11, 21, 31, 41, 51];
console.log("Asi se ve la lista antes de la eliminacion");
console.log(numeros.toString());
//esta parte es la que elimina el primer numero de la lista, para esto se crea un nuevo arreglo y se le asigna a cada posicion el valor del arreglo original, pero empezando desde la segunda posicion

let n = numeros.length;
let pos = 2;
let nuevo = []
for (let j = 0; j<n -1; j++ ){
    if (j < pos){
        nuevo[j] = numeros[j];
    }else{
        nuevo[j] = numeros[j + 1];
    }
    
}
console.log("Despues de la eliminacion queda: ");
console.log(nuevo.toString());

File: python/EliminacionPersonalizada.py
inputArr = [11, 21, 31, 41, 51, 61]
print("Antes de la eliminacion, el arreglo es: ")
for j in range(len(inputArr)):
    print(inputArr[j], end=" ")
n = len(inputArr)
pos = int(input("Ingrese la posicion del elemento a eliminar: "))
for i in range((pos - 1 ), n - 1):
    inputArr[i] = inputArr[i + 1]

n-=1
inputArr = inputArr[:n]

    
print("\n Despues de la eliminacion, el array es: ")

for j in range(len(inputArr)):
    print(inputArr[j], end=" ")
    